internal/docs/public/index.html

```diff
@@ -23,5 +23,5 @@
 
   <body>
     <div id="root"></div>
-  <script type="text/javascript" src="main.9d0573e3d9566df71fc9.js"></script></body>
+  <script type="text/javascript" src="main.71e28e13506319cf9036.js"></script></body>
```

Comment:
Not sure if these changes are good or bad.

Reply:
not needed afaik, this should be git ignored

Reply:
Ok cool. I will add that in a followup.